spec/src/main/asciidoc/microprofile-openapi-spec.adoc

```diff
@@ -180,41 +221,231 @@ public Response findPetsByStatus(...) { ... }
 [source, yaml]
 ----
 /pet/findByStatus:
-    get:
+  get:
     summary: Finds Pets by status
     description: Multiple status values can be provided with comma separated strings
     operationId: findPetsByStatus
 ----
 
+.Sample 2 - Operation with different responses
+[source,Java]
+----
+@GET
+@Path("/{username}")
+@Operation(summary = "Get user by user name",
+           responses = {
+                @APIResponse(description = "The user",
+                             content = @Content(mediaType = "application/json",
+                             schema = @Schema(implementation = User.class))),
+                @APIResponse(responseCode = "400", description = "User not found")})
+public Response getUserByName(
+        @Parameter(description = "The name that needs to be fetched. Use user1 for testing. ", required = true) @PathParam("username") String username)
+ {...}
+----
 
-===== APIResponse
+.Output for Sample 2
+[source, yaml]
+----
+/user/{username}:
+    get:
+      summary: Get user by user name
+      operationId: getUserByName
+      parameters:
+      - name: username
+        in: path
+        description: 'The name that needs to be fetched. Use user1 for testing. '
+        required: true
+        schema:
+          type: string
+      responses:
+        default:
+          description: The user
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        400:
+          description: User not found
+----
 
-TODO: Detailed usage description of annotation (issue #10)
 
-.Sample 1
+===== RequestBody
+
+
+.Sample 1 - Simple RequestBody
 [source,Java]
 ----
-
+@POST
+        @Path("/user")
+        @Operation(summary = "Create user",
+                   description = "This can only be done by the logged in user.")
+        public Response methodWithRequestBodyAndTwoParameters(
+                @RequestBody(description = "Created user object", required = true,
+                             content = @Content(
+                             schema = @Schema(implementation = User.class))) User user,
+                @QueryParam("name") String name, @QueryParam("code") String code)
+                { ... }
```

Comment:
Indentation of this block is wrong - everything except `@POST` should be de-dented one level.  In other words, `@Path`, `@Operation`, and `public Response...` should all be at the same level as `@POST`.

Reply:
+1